### DISCUSSION TITLE
Unable to update to 1.26 from 1.25.2

### DISCUSSION THREAD

**Discussion by Xanderlicious** [dani-garcia/vaultwarden#2830] [Q&A]
Trying to update like I have always done through portainer.  Duplicate/edit and then deploy

Getting the below errors

/--------------------------------------------------------------------\

| Starting Vaultwarden |

| Version 1.25.2 |

|--------------------------------------------------------------------|

| This is an *unofficial* Bitwarden implementation, DO NOT use the |

| official channels to report bugs/features, regardless of client. |

| Send usage/configuration questions or feature requests to: |

| https://vaultwarden.discourse.group/ |

| Report suspected bugs/issues in the software itself at: |

| https://github.com/dani-garcia/vaultwarden/issues/new |

\--------------------------------------------------------------------/

[INFO] No .env file found.

[DEPRECATED]: `SMTP_SSL` or `SMTP_EXPLICIT_TLS` is set. Please use `SMTP_SECURITY` instead.

[2022-10-16 09:56:38.202][vaultwarden::api::notifications][INFO] Starting WebSockets server on 0.0.0.0:3012

[2022-10-16 09:56:38.203][start][INFO] Rocket has launched from http://0.0.0.0:80

[2022-10-16 09:59:17.426][vaultwarden][INFO] Exiting vaultwarden!

[2022-10-16 09:59:17.426][rocket::server][WARN] Received SIGTERM. Requesting shutdown.

[2022-10-16 09:59:17.426][vaultwarden::api::notifications][INFO] Shutting down WebSockets server!

[2022-10-16 09:59:17.426][vaultwarden][INFO] Vaultwarden process exited!

I think there is an issue with some of my environment settings but not sure which. Tried changing the SSL options for e-mail but this still doesn't allow the container to start

Here are my env settings

DOMAINhttps://bitwarden.domain.co.uk
INVITATIONS_ALLOWEDtrue
PATH/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
ROCKET_ADDRESS0.0.0.0
ROCKET_ENVstaging
ROCKET_PORT80
ROCKET_PROFILErelease
ROCKET_WORKERS10
SIGNUPS_ALLOWEDtrue
SMTP_AUTH_MECHANISMPlain
SMTP_EXPLICIT_TLStrue
SMTP_FROMexamplemail@gmail.com
SMTP_HOSTsmtp.gmail.com
SMTP_PASSWORDwouldntyouliketoknow
SMTP_PORT465
SMTP_SSLtrue
SMTP_USERNAMEeamplemail@gmail.com
WEBSOCKET_ENABLEDtrue

Any help here is appreciated

Alex

**Comment by BlackDex** [comment]
I still see v1.25.2 instead of 1.26.0 during the startup.
So, the image wasn't updated correctly.

Also, something from outside is shutting down Vaultwarden, since it received a SIGTERM command, that is not something Vaultwarden does by it self.

So, first check if portainer did actually pulled in the new image.

**Comment by Xanderlicious** [comment]
> I still see v1.25.2 instead of 1.26.0 during the startup.
> So, the image wasn't updated correctly.
> 
> Also, something from outside is shutting down Vaultwarden, since it received a SIGTERM command, that is not something Vaultwarden does by it self.
> 
> So, first check if portainer did actually pulled in the new image.
> 

Thanks for the reply,. I have edited my question to include my env settings. Do these look right to you?

I'm going to delete the image and container and then try setting it up again from scratch

**Reply by BlackDex** [reply]
Looks ok, you might want to change the SMTP SSL Stuff like reported and add `SMTP_SECURITY=starttls`
And, it's also best to remove the following: `ROCKET_ENV`, `ROCKET_PROFILE`, `ROCKET_WORKERS`.
